lib/render/game.php

```diff
@@ -212,7 +212,7 @@ function RenderLinkToGameForum($gameTitle, $gameID, $forumTopicID, $permissions
         echo "<a class='info-button' href='/viewtopic.php?t=$forumTopicID'><span>💬</span>Official forum topic</a>";
     } else {
         if ($permissions >= Permissions::Developer) {
-            echo "<a class='info-button' href='/request/game/generate-forum-topic.php?g=$gameID' onclick='return confirm(\"Are you sure you want to create the official forum topic for this game?\")'><span>💬</span>Create the official forum topic for $gameTitle</a>";
+            echo "<a class='info-button' href='/request/game/generate-forum-topic.php?g=$gameID&f=0' onclick='return confirm(\"Are you sure you want to create the official forum topic for this game?\")'><span>💬</span>Create the official forum topic for $gameTitle</a>";
```

Comment:
If a non-null default is expected, shouldn't it be defined in the `requestInputQuery` processing the request?

Reply:
Updated the `requestInputQuery` to have a default of 0 and removed the `&f=0` form the above code.